src/Fabfuel/Prophiler/Adapter/Interop/Container/Container.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * @author @shochdoerfer <S.Hochdoerfer@bitExpert.de>
+ * @created 18.10.15, 12:14
+ */
+namespace Fabfuel\Prophiler\Adapter\Interop\Container;
+
+use Fabfuel\Prophiler\Adapter\AdapterAbstract;
+use Fabfuel\Prophiler\ProfilerInterface;
+use Interop\Container\ContainerInterface;
+
+class Container extends AdapterAbstract implements ContainerInterface
```

Comment:
Please add a unit test for your adapter and please have a look at the Scrutinizer issues, mentioned here:
https://scrutinizer-ci.com/g/fabfuel/prophiler/inspections/c8e11dda-bae4-4a30-a391-9da6d35e69e4/issues/